src/sentry/db/models/base.py

```diff
@@ -66,15 +76,21 @@ def __setstate__(self, state):
         self._update_tracked_data()
 
     def __get_field_value(self, field):
+        if isinstance(type(field).__dict__.get(field.attname), DeferredAttribute):
+            return DEFERRED
         if isinstance(field, models.ForeignKey):
             return getattr(self, field.column, None)
-        return getattr(self, field.name, None)
+        return getattr(self, field.attname, None)
 
     def _update_tracked_data(self):
         "Updates a local copy of attributes values"
         if self.id:
             data = {}
             for f in self._meta.fields:
+                # XXX(dcramer): this is how Django determines this (copypasta from Model)
```

Comment:
fwiw i dont think this did anything, as it definitely didnt resolve the original issue I saw